src/components/news-card.tsx

```diff
@@ -0,0 +1,64 @@
+import { formatDistance } from "date-fns";
+import { pl } from "date-fns/locale";
+import React from "react";
+
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+import type { NewsPost } from "@/lib/types";
+
+import { Card, CardContent, CardHeader, CardTitle } from "./ui/card";
+
+export function NewsCard({ post }: { post: NewsPost }) {
+  const today = new Date();
+
+  const getFriendlyStringDate = (date: string) => {
+    const dateObject = new Date(date);
+
+    const timePassed = formatDistance(dateObject, today, {
+      addSuffix: true,
+      locale: pl,
+    });
+
+    return timePassed;
+  };
+
```

Comment:
lepiej wyodrębnić żeby nie powtarzac dwa razy

```suggestion

  const createdText = `Utworzono ${getFriendlyStringDate(post.date_created)}`;

```